geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/index/z2/XZ2WritableIndex.scala

```diff
@@ -43,7 +42,7 @@ trait XZ2WritableIndex extends AccumuloWritableIndex {
     val sfc = XZ2SFC(sft.getXZPrecision)
     val sharing = sft.getTableSharingBytes
     (wf: AccumuloFeature) => {
-      val mutation = new Mutation(getRowKey(sfc, sharing)(wf))
+      val mutation = new Mutation(getRowKey(sfc, sharing, sft)(wf))
```

Comment:
probably makes sense to pull out the calls to split arrays and num splits and pass them into the method directly  (same way we do with table sharing bytes) to save us having to look it up for each feature